### ISSUE TITLE
linux6.6 i386 boot failed

### ISSUE THREAD

**Issue by naY9yjoS6ZqhOd35sIFH** [void-linux/void-packages#47514]
### Is this a new report?

Yes

### System Info

Void 6.1.63_1

### Package(s) Affected

linux6.6

### Expected behaviour

boot and show dmesg to serial console

### Actual behaviour

stuck after loading vmlinuz

### Steps to reproduce

1.install linux6.6 dracut and grub from xbps-install
2.generate initramfs from dracut
3.using grub-mkconfig to add grub menu
4.reboot and stuck at loading

**Comment by mtboehlke** [comment]
I was able to reproduce on my i686 machine.  After decompressing the kernel successfully, boot hangs on message: `Booting the kernel`
I use syslinux, so probably not a grub problem.

Linux 6.5.13 boots fine.

**Comment by naY9yjoS6ZqhOd35sIFH** [comment]
and 6.5 is eol now 
@Duncaen

**Comment by leahneukirchen** [comment]
Are you sure your /boot isn't full and the initrd can be read completely? (try `lsinitrd`)

**Comment by mtboehlke** [comment]
No, at least that wasn't it for me.  My /boot partition is less than half full.  I usually run an lts kernel.  I installed linux6.6 to see if I had the same issue, and installed 6.5 afterwards and was able to boot, so I am not running into a space issue.  No error messages during initramfs generation, and lsinitrd succeeds and looks normal.  I just tested 6.6.4 just in case, but still stuck.  This issue seems to be specific to the 6.6 series.
